### PR TITLE
Pilot Notebook

### PR DESCRIPTION
**Pull request by claalmve**
**Notebook attached to SCE pilot backend**
This notebook should be run in conjunction with https://github.com/cal-adapt/climakitae/pull/352 to see the full functionality of the `cava_data` function in `vulnerability.py`.

### CODE REVIEW

File: collaborative/IOU/vulnerability_assessment.ipynb [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "153b50c3-5e8c-4f4c-b7e2-1839ae0c462f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Step 1: Import locations\n",
+    "Now we'll read in point-based locations that we want to retrieve data for. For custom inputs, there are two options: (1) Input a single pair of latitude - longitude values; and (2) Import a csv file of locations that will each run. In the code below we show what each option looks like. \n",
```
Comment (Tianchi-Liu) [inline collaborative/IOU/vulnerability_assessment.ipynb:46]: Backend doesn’t seem to support single values yet.

File: collaborative/IOU/vulnerability_assessment.ipynb [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    "# your_lat = LAT\n",
+    "# your_lon = LON"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c435c50f-dec5-4440-9bd6-e5207a317f92",
+   "metadata": {},
+   "source": [
+    "To import your own custom locations, we recommend putting your csv file in the same folder as this notebook for ease:\n",
```
Comment (Tianchi-Liu) [inline collaborative/IOU/vulnerability_assessment.ipynb:68]: A dedicated folder for such files would be nice to have, similarly for exported data, to keep the folder organized. 

Showing example_file.csv as an example may also help.

File: collaborative/IOU/vulnerability_assessment.ipynb [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+   "source": [
+    "Now, with the above desired points, we can pass in a series of arguments into the below `cava_data` function to retrieve the desired raw data and/or computed data for your CAVA analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85760d97",
+   "metadata": {},
+   "source": [
+    "Below are examples of some of the available selections for the `variable`, `metric_calc`, `ssp_data`, and `export_method` parameters."
```
Comment (Tianchi-Liu) [inline collaborative/IOU/vulnerability_assessment.ipynb:129]: The parameters could use more explanation for general users, especially export_method.

File: collaborative/IOU/vulnerability_assessment.ipynb [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+   "id": "234dd10c-b0ee-49a8-929d-a8c692cbfa9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = cava_data(\n",
+    "    example_locs.iloc[:2],\n",
+    "    time_start_year=2030,\n",
+    "    time_end_year=2050,\n",
+    "    units=\"degF\",\n",
+    "    downscaling_method=\"Dynamical\",  # default for now ## mandatory\n",
```
Comment (Tianchi-Liu) [inline collaborative/IOU/vulnerability_assessment.ipynb:158]: Statistical doesn’t seem to work yet.

Comment (claalmve) [reply collaborative/IOU/vulnerability_assessment.ipynb:158]: Yes, this is intentional for this version. I will be showcasing that this works with Dynamical data and that Statistical data will be included in phase 3 of development.

Comment (claalmve) [reply collaborative/IOU/vulnerability_assessment.ipynb:158]: I can add some helper text in the function to make sure the users know this too.

**Pull request merged**